ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcInputFormat.java

```diff
@@ -122,6 +122,7 @@ public class VectorizedOrcInputFormat extends FileInputFormat<NullWritable, Vect
     public boolean next(NullWritable key, VectorizedRowBatch value) throws IOException {
 
       try {
+        value.reset();
```

Comment:
could you please wrap with the `try` just `reader.nextBatch`

Reply:
````
        value.reset();
        // Check and update partition cols if necessary. Ideally, this should be done
        // in CreateValue as the partition is constant per split. But since Hive uses
        // CombineHiveRecordReader and
        // as this does not call CreateValue for each new RecordReader it creates, this check is
        // required in next()
        if (addPartitionCols) {
          if (partitionValues != null) {
            rbCtx.addPartitionColsToBatch(value, partitionValues);
          }
          addPartitionCols = false;
        }
      try {
        if (!reader.nextBatch(value)) {
          return false;
        }
      } catch (Exception e) {
        throw new RuntimeException(e);
      }
````